src/MasterSite.php

```diff
@@ -122,7 +122,7 @@ protected function hooks() {
 		// Load the editor scripts only enqueuing editor scripts while in context of the editor.
 		add_action( 'enqueue_block_editor_assets', [ $this, 'enqueue_editor_assets' ] );
 		// Load main theme assets before any child theme.
-		add_action( 'wp_enqueue_scripts', [ PublicAssets::class, 'enqueue' ] );
+		add_action( 'wp_enqueue_scripts', [ PublicAssets::class, 'enqueue' ], 0 );
```

Comment:
I suppose this means the child themes that use priority 10 or less should be checked as their loading order will change?

Reply:
Yep, but we already have problems with those (cf [dev japan](https://www-dev.greenpeace.org/japan/)), that's why I'm working on this